lenovo/thinkpad/x1/7th-gen/audio.nix

```
@@ -0,0 +1,13 @@
+{ lib, pkgs, ... }:
+{
+  # This can be removed when the default kernel is at least version 5.5
+  # https://github.com/NixOS/nixpkgs/pull/86168
+  boot.kernelPackages = lib.mkDefault pkgs.linuxPackages_5_6;
```

How about

```suggestion
  boot.kernelPackages = mkIf (stdenv.lib.versionOlder kernel.version "5.6") lib.mkDefault pkgs.linuxPackages_latest;
```

This way if our default kernel is not 5.6 and the user has not changed the default kernel, we use the latest kernel, which is 5.6 at the time.

---

Should be

---

Thanks! I adopted that with one change: use 5.6 instead of latest, since that's the version that has been tested.